src/app/child-dev-project/attendance/model/attendance-day.ts

```diff
@@ -16,16 +16,11 @@
  */
 
 import { DatabaseField } from "../../../core/entity/database-field.decorator";
+import { AttendanceStatus } from "./attendance-status";
 
-export enum AttendanceStatus {
-  UNKNOWN = "?",
-  HOLIDAY = "H",
-  ABSENT = "A",
-  PRESENT = "P",
-  LATE = "L",
-  EXCUSED = "E",
-}
-
+/**
+ * @deprecated Use Event entities instead of the embedded AttendanceDay and AttendanceMonth
```

Comment:
To finish this epic, this class should be completely removed.

Reply:
agreed